imports/ui/QuestionEditItem.jsx

```diff
@@ -81,14 +72,14 @@ export class QuestionEditItem extends Component {
           break
       }
     } else { // if adding new question
-      this.state = _.extend({}, DEFAULT_STATE)
+      this.state = _.extend({}, defaultQuestion)
```

Comment:
defaultQuestion has a "creator" property in it, so make sure there is no conflict. 

 I think that we want to ultimately have something like

this.sate.question=defaultQuestion

so let's file that as an issue and worry about it later.

Reply:
Would it be a good idea to get rid of the owner property and just have the creator property (or vice versa)?

Reply:
Nope! The reason for having the two is when we want a TA to moderate the questions submitted to the course by a student. The student is the creator, but the TA would take ownership of the question when approving it (so that they can modify it). Thus, when questions are created, owner and creator should be the same person that create the question, but when one copies the question, then the owner could change. At least, that was the thinking...